references.bib

```diff
@@ -158,6 +158,16 @@ @inproceedings{Wails2024a
 	discussion_url = {https://github.com/net4people/bbs/issues/312},
 }
 
+@inproceedings{Niere2023a,
+	author = {Niklas Niere and Sven Hebrok and Juraj Somorovsky and Robert Merget},
+	title = {Poster: Circumventing the GFW with TLS Record Fragmentation},
```

Comment:
Curly brackets are useful to preserve the capitalization of names:
```suggestion
	title = {Poster: Circumventing the {GFW} with {TLS} Record Fragmentation},
```

Reply:
Thanks for the hint, added!